debezium-server/pom.xml

```diff
@@ -0,0 +1,228 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>io.debezium</groupId>
+        <artifactId>debezium-parent</artifactId>
+        <version>1.2.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>debezium-server</artifactId>
+    <name>Debezium Standalone Quarkus Server</name>
+    <packaging>jar</packaging>
+
+    <properties>
+        <version.kinesis>2.13.13</version.kinesis>
+
+        <!-- Assembly configuration -->
+        <assembly.descriptor>server-distribution</assembly.descriptor>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-bom-deployment</artifactId>
+                <version>${quarkus.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+
+            <!-- Target systems -->
+            <dependency>
```

Comment:
What's the license? Can we package this?

Reply:
@gunnarmorling It is ASL so we should be good to go.